pinot-segment-local/src/main/java/org/apache/pinot/segment/local/upsert/Distribution.java

```diff
@@ -0,0 +1,71 @@
+package org.apache.pinot.segment.local.upsert;
+
+import java.util.Arrays;
+import java.util.Random;
+import java.util.SplittableRandom;
+import java.util.function.DoubleSupplier;
+import java.util.function.LongSupplier;
+import org.apache.commons.lang3.tuple.Pair;
+
+
+public enum Distribution {
```

Comment:
it seems that this class is just the same than `org.apache.pinot.perf.Distribution`. If we really need it here, I would then remove it from perf, which I guess already depends on pinot-segment-local